src/js/service/ImportService.js

```diff
@@ -0,0 +1,126 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
+(function () {
+  var ns = $.namespace('pskl.service');
+
+  /**
+   * @param {!PiskelController} piskelController
+   * @param {!PreviewController} previewController
+   * @constructor
+   */
+  ns.ImportService = function (piskelController, previewController) {
+    this.piskelController_ = piskelController;
+    this.previewController_ = previewController;
+  };
+
+  ns.ImportService.prototype.init = function () {
```

Comment:
:unamused: The init function for _every_ module seems to be a pattern in this codebase (see empty init methods in `ImageUploadService` and `PiskelController`), though I'm happy to deviate from it for our own code.